configs/base_env.rc

```diff
@@ -18,7 +18,7 @@ _sentry_db_host = localhost
 # _sentry_db_password =
 # Replace with your admin email if necessary
 assembl.admin_email = webmaster@%(public_hostname)s
-
+assembl.generic_auth_errors = false
```

Comment:
Let's rename to `generic_errors` since this could affect more than auth (in the future)